scripts/liquidation_bot/index.ts

```diff
@@ -108,19 +120,23 @@ async function main() {
       loops = 0;
     }
 
+    // TODO: Remove this line entirely
     const currentBlockNumber = await hre.ethers.provider.getBlockNumber();
 
-    googleCloudLog(LogSeverity.INFO, `currentBlockNumber: ${currentBlockNumber}`);
+    googleCloudLog(LogSeverity.INFO, `currentBlockNumber: ${currentBlockNumber} [checked=${lastCheckedBlockNumber}]`);
 
     if (currentBlockNumber !== lastBlockNumber) {
```

Comment:
This check should now be checking that `currentBlockNumber != lastCheckedBlockNumber` right?